internal/emlb/emlb.go

```diff
@@ -76,7 +76,7 @@ type EMLB struct {
 	client         *lbaas.APIClient
 	metro          string
 	projectID      string
-	tokenExchanger *TokenExchanger
```

Comment:
these changes are bleeding over from #778 and are being reviewed there.

Reply:
I've removed the extra commits.